locations/spiders/iron_mountain.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+
+from locations.items import GeojsonPointItem
+
+
+class IronMountainSpider(scrapy.Spider):
+    name = "iron_mountain"
+    start_urls = (
+        'https://locations.ironmountain.com/',
+    )
+    
+    def parse(self, response):
+        urls = response.xpath('//*[@id="browse-expand"]/div/div[1]/div/li/div/a/@href').extract()
+        for url in urls:
+            yield scrapy.Request(url=url, callback=self.parse_list,meta={'country': "USA"})
+        urls = response.xpath('//*[@id="browse-expand"]/div/div[2]/div/li/div/a/@href').extract()
+        for url in urls:
+            yield scrapy.Request(url=url, callback=self.parse_list,meta={'country': "Canada"})
+
+    def parse_list(self,response):
+        urls = response.xpath('//*[@id="main-container"]/div[2]/div[3]/div[1]/div/div[2]/div[2]/ul/li/div/a/@href').extract()
+        for url in urls:
+            yield scrapy.Request(url=url, callback=self.parse_location,meta={'country': response.meta.get("country")})
+
+    def parse_city(self,response):
+        urls = response.xpath('//*[@id="main-container"]/div[2]/div[3]/div[1]/div/div[2]/div[2]/ul/li/a/@href').extract()
+        for url in urls:
+            yield scrapy.Request(url=url, callback=self.parse_location,meta={'country': response.meta.get("country")})
+    
+    def parse_location(self,response):
+        data = re.sub("'","",response.xpath('//*[@id="gmap"]/script/text()').extract()[0])
+        data = data.replace("false","False")
+        data = data.replace("true","True")
+        data = data.replace("null","None")
+        data = data.replace("RLS.defaultData = ","")
+        data = data.replace(";","")
+        address = eval("{"+str.join(",",eval(data)["markerData"][0]["info"].replace("<\\/div>","").split(",")[1:]))
+        yield GeojsonPointItem(**{
```

Comment:
No need to create a dict and then `**` it. Just pass the args straight in like so:

```python
yield GeojsonPointItem(
  addr_full="..."
```

Reply:
Okay I will change that.